v2/internal/pkg/cli/executor.go

```diff
@@ -138,6 +138,7 @@ type ExecutorSchema struct {
 	MirrorUnArchiver    archive.UnArchiver
 	MakeDir             MakeDirInterface
 	Delete              delete.DeleteInterface
+	SynchedTimeStamp    string
```

Comment:
Bikeshedding: can we find a more descriptive name for this? What about
```suggestion
	MirrorStartTimestamp    string
```